acceptance_tests/features/steps/load_seft_collection_instruments.py

```diff
@@ -0,0 +1,20 @@
+from behave import given, when, then
+
+from acceptance_tests.features.pages import collection_exercise_details
+
+
+@given('the 201803 collection exercise for the QIFDI survey has been created')
```

Comment:
Is there a reason for choosing QIFDI? For the other tasks we use QBS 1803, it might be good to have consistency across the different tests

Reply:
I don't think we should choose the same survey across the tests, I think it's fine to select different ones.

Reply:
That's a good point about consistency. I chose QIFDI as it was used in the wireframe examples for the story but I'm happy to change it so it's aligned with the other tests.